packages/blade/src/components/Table/docs/APIStories/TableNesting.stories.tsx

```diff
@@ -0,0 +1,386 @@
+    merchantId: 'rzp_merchant_amazon',
+    totalAmount: 43000,
+    transactionCount: 1247,
+    successRate: 66.7,
+    status: 'Active',
+    children: [
+      {
+        id: 'txn_1',
+        paymentId: 'rzp_pay_amazon_001',
+        amount: 10000,
+        date: '2024-01-15',
+        method: 'Credit Card',
+        customerEmail: 'customer@example.com',
+        status: 'Completed',
+      },
+      {
+        id: 'txn_2',
+        paymentId: 'rzp_pay_amazon_002',
+        amount: 16000,
+        date: '2024-01-15',
+        method: 'UPI',
+        customerEmail: 'user@gmail.com',
+        status: 'Completed',
+      },
+      {
+        id: 'txn_3',
+        paymentId: 'rzp_pay_amazon_003',
+        amount: 7000,
+        date: '2024-01-14',
+        method: 'Net Banking',
+        customerEmail: 'buyer@yahoo.com',
+        status: 'Failed',
+      },
+    ],
+  },
+  {
+    id: 'merchant_2',
+    merchantName: 'Flipkart',
+    merchantId: 'rzp_merchant_flipkart',
+    totalAmount: 24500,
+    transactionCount: 983,
+    successRate: 100.0,
+    status: 'Active',
+    children: [
+      {
+        gatewayUsed: 'Razorpay',
+        totalAmount: 18000,
+        transactionCount: 790,
+        successRate: 100.0,
+      },
+    ],
+  },
+  {
+    id: 'merchant_3',
+    merchantName: 'Zomato',
+    merchantId: 'rzp_merchant_zomato',
+    totalAmount: 1450,
+    transactionCount: 756,
+    successRate: 50.0,
+    status: 'Active',
+    children: [
+      {
+        id: 'txn_z1',
+        paymentId: 'rzp_pay_zomato_001',
+        amount: 450,
+        date: '2024-01-15',
+        method: 'UPI',
+        customerEmail: 'foodie@zomato.com',
+        status: 'Completed',
+      },
+      {
+        id: 'txn_z2',
+        paymentId: 'rzp_pay_zomato_002',
+        amount: 1000,
+        date: '2024-01-15',
+        method: 'Credit Card',
+        customerEmail: 'customer@food.com',
+        status: 'Failed',
+      },
+    ],
+  },
+];
+
+// Custom slide down animation component
+const SlideDown = ({ children }: { children: React.ReactElement }): React.ReactElement => {
+  const { theme } = useTheme();
+  const movePx = makeSpace(theme.spacing[3]); // 8px offset from above
+
+  const slideDownVariants: MotionVariantsType = {
+    initial: {
+      opacity: 0,
+      transform: `translateY(-${movePx})`, // Start from above (negative Y)
+    },
+    animate: {
+      opacity: 1,
+      transform: `translateY(${makeSpace(theme.spacing[0])})`, // End at 0
+      transition: {
+        duration: msToSeconds(theme.motion.duration.quick),
+        ease: cssBezierToArray(castWebType(theme.motion.easing.entrance)),
+      },
+    },
+    exit: {
+      opacity: 0,
+      transform: `translateY(-${movePx})`, // Exit upward
+      transition: {
+        duration: msToSeconds(theme.motion.duration.quick),
+        ease: cssBezierToArray(castWebType(theme.motion.easing.exit)),
+      },
+    },
+  };
+
+  return (
+    <BaseMotionEntryExit
```

Comment:
This component is not exposed to consumers so it won't work when they copy-paste code

Did you mean to use `<Slide />` component? Also do animations have to be on consumer-side?